src/main/scala/rise/autotune/autoTuningErrors.scala

```diff
@@ -0,0 +1,12 @@
+package rise.autotune
+
+sealed trait AutoTuningErrorLevel
+
+case object NO_ERROR extends AutoTuningErrorLevel
+case object CONSTRAINTS_ERROR extends AutoTuningErrorLevel
+case object SUBSTITUTION_ERROR extends AutoTuningErrorLevel
+case object CODE_GENERATION_ERROR extends AutoTuningErrorLevel
+case object COMPILATION_ERROR extends AutoTuningErrorLevel
+case object EXECUTION_ERROR extends AutoTuningErrorLevel
+
+case class AutoTuningError(errorLevel: AutoTuningErrorLevel, message:Option[String])
```

Comment:
maybe the file can just be named `autotune/errors.scala`? no need to repeat autotuning. same for the other files.

Reply:
I removed the *autoTuning* prefix from the files.